### ISSUE TITLE
Richtlijnen: Zeggen we in onze richtlijnen 19px of 18.5px voor grenswaarde contrast 3:1?

### ISSUE THREAD

**Issue by rianrietveld** [nl-design-system/documentatie#1923]
Vraag Rozerin in Slack:

In onze [contrast checker tool](https://nldesignsystem.nl/contrast/?background-color=%23ffffff&color=%23000333) geven we aan:
Je mag de kleuren gebruiken voor vette tekst groter dan 18.5px, als je wilt voldoen aan WCAG 2.2 AA niveau.
In onze [richtlijnen](https://arc.net/l/quote/swtwfqry) staat echter:
3:1 contrast voor vette letters (vet en groter of gelijk aan 19 pixels)
Moet vette tekst minstens 18.5px of 19px zijn? Kunnen we dit gelijktrekken om consistentie tussen onze tool en richtlijnen te waarborgen?

Argumentatie Robbert:
DigiD gebruikt al jaren 18.5 of 18.667px voor de wit op oranje letters van de primary button. Als we onze tekst aanpassen, dan heeft enkele effecten:
- Aline zal geneigd zijn de font size naar 19px aan te passen
- het visuele verschil tussen de primary en de secondary button wordt groter
- we zullen voor consistentie ook 19px moeten aanhouden in de theme validator
- we zullen dan een foutmelding geven bij thema's met 18,5px
- we gaan discussie krijgen dat we valide thema's afkeuren

## Actie
Richtlijnen, acceptatiecriteria componenten en de WCAG pagina's waar 19px staat aanpassen naar 18,5 pixels.

## Acceptatiecriteria
De eisen voor kleurcontrast zijn gelijk voor de:
- [ ] de richtlijnen
- [ ] acceptatiecriteria componenten
- [ ] de WCAG-pagina's
- [ ] en de contrast checker

**Comment by Robbert** [comment]
Ik heb me wel eens afgevraagd of 18.5px heel onscherpe letters oplevert op Windows wanneer je geen retina-scherm hebt. En of je dat om voldoende contrast te krijgen met scherpe letters met minder anti-aliasing je dan wel moet afronden naar 19px

En of je dat ongeveer zo moet oplossen:

```css
:root {
  --nl-button-size: 19px;
}

@media screen and (min-device-pixel-ratio: 2) {
  :root {
    --nl-button-size: 18.5px;
  }
}
```
